spec/tests/yaml/swiftnav/sbp/test_bootload.yaml

```diff
@@ -6,11 +6,20 @@ tests:
 
 - msg:
     fields:
-      handshake: 118
+      handshake: [118, 49, 46, 50] # 'v1.2'
```

Comment:
Hmmm, I suppose this is fine for the time being. It's more a problem that we aren't being consistent with our schema (null terminated strings). I'll probably follow up at some point with making this parser conditional.